.github/workflows/cleanup-firebase.yml

```diff
@@ -0,0 +1,29 @@
+name: Cleanup Firebase Metadata
+
+on:
+    schedule:
```

Comment:
Currently, this job is scheduled to run weekly. We can add additional triggers or modify the frequency of the job if needed.

Reply:
only `schedule` should be suffice for now